packages/lesswrong/components/revisions/TagPageRevisionSelect.tsx

```diff
@@ -56,15 +55,13 @@ const TagPageRevisionSelect = ({ classes }: {
         totalCount={totalCount}
       />
       {revisions.map((rev, i)=> {
```

Comment:
Just to check my understanding, this check is no longer necessary because of the changes in the diffResolver?

Reply:
Yes, we now actively want to show the first revision, which is what this check was filtering out.